docs/consume-packages/consuming-packages-authenticated-feeds.md

```diff
@@ -3,98 +3,106 @@ title: Consuming packages from authenticated feeds
 description: Consuming packages from authenticated feeds in all NuGet client scenarios
 author: nkolev92
 ms.author: nikolev
-ms.date: 02/28/2020
+ms.date: 12/22/2023
 ms.topic: conceptual
 ---
 
 # Consuming packages from authenticated feeds
 
-In addition to the nuget.org [public feed](https://api.nuget.org/v3/index.json), NuGet clients have the ability to interact with file feeds and private http feeds.
+Many NuGet operations, such as restore and install, require communication with one or more package sources, which [can be configured in *nuget.config* files](../reference/nuget-config-file.md#packagesources).
+For HTTP feeds, NuGet will with make an unauthenticated request, and if the server responds with an HTTP 401 response, NuGet will search for credentials in the following order:
 
+1. [An environment variable `NuGetPackageSourceCredentials_{name}`](#credentials-in-environment-variables).
+1. [Credentials in *nuget.config* files](#credentials-in-nugetconfig-files).
+1. [Use a NuGet credential provider, if your package source provides one](#credential-providers).
 
-To authenticate with private http feeds, the 2 approaches are:
+> [!NOTE]
+> We recommend using a credential provider when possible.
+> Secrets will not be saved in the *nuget.config* file, reducing risk of accidentally leaking secrets.
+> Additionally, it typically reduces the number of places you need to update when a credential expires or changes.
+> If the credential provider supports single sign on, it may reduce the number of time you need to log in, or the number of places that credentials need to be saved.

```

Comment:
```suggestion
> If the credential provider supports single sign-on, it may reduce the number of times you need to login, or the number of places that credentials need to be saved.

```